modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/tests/RoleUtils.spec.ts

```diff
@@ -137,3 +139,80 @@ it('Assert roles in User Notification', () => {
 		},
 	]);
 });
+
+it('verify that handleMultiSelectRoleItemsChange generates new recipients in the correct data structure', () => {
```

Comment:
everything looks great, you just forgot to rename the `RoleUtils.spec.ts` to `RolesUtil.spec.ts` following your previous changes